plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java

```diff
@@ -361,6 +367,7 @@ public void widgetSelected(SelectionEvent e) {
       @Override
       public void run() {
         refreshProjectsForSelectedCredential();
+        refreshProjectsButton.setEnabled(accountSelector.getSelectedCredential() != null);
```

Comment:
When databinding is added, the `accountSelector`'s `selectAccount()` is called and that will in turn call this method, so it will initialize the enablement.
However we add databinding for this enablement instead of setting it in the event handler.